scripts/plot.py

```diff
@@ -4,13 +4,17 @@
 matplotlib.use('Agg')
 
 import os
+import sys
 import argparse
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.patches as patches
 from matplotlib.backends.backend_pdf import PdfPages
 
 def get_cpu_name():
+    if 'darwin' == sys.platform:
+        return os.popen("sysctl -n machdep.cpu.brand_string").read().strip()
```

Comment:
Could you please add this in a single commit: "darwin: script: implement get_cpu_name() using sysctl"?